50-qmk.rules: Move udev rules from documentation into a file (#10664)
qmk/qmk_firmware: docs/faq_build.md

### Linux `udev` Rules

> On Linux, you'll need proper privileges to communicate with the bootloader device. You can either use `sudo` when flashing firmware, or place a udev rules file in [`/etc/udev/rules.d/50-qmk.rules`](https://github.com/qmk/qmk_firmware/tree/master/util/udev/50-qmk.rules).

Reviewer: ```suggestion
On Linux, you'll need proper privileges to communicate with the bootloader device. You can either use `sudo` when flashing firmware (not recommended), or place [this file](https://github.com/qmk/qmk_firmware/tree/master/util/udev/50-qmk.rules) into `/etc/udev/rules.d/`.
```
